src/Servers/IIS/IISIntegration/src/IISHostingStartup.cs

```diff
@@ -7,8 +7,18 @@
 
 namespace Microsoft.AspNetCore.Server.IISIntegration
 {
+    /// <summary>
+    /// The <see cref="IHostingStartup"/> to add IISIntegration to apps.
+    /// </summary>
     public class IISHostingStartup : IHostingStartup
```

Comment:
Could we add a remark that says this API isn't mean to be used by user code? We don't expect users to reference this do we?